beetmoverscript/templates/fennec_candidates.yml

```diff
@@ -0,0 +1,114 @@
+      destinations:
+        - {{ platform }}/{{ locale }}/fennec-{{ version }}.{{ locale }}.android-arm.common.tests.zip
+    target.cppunittest.tests.zip:
+      s3_key: fennec-{{ version }}.{{ locale }}.android-arm.cppunittest.tests.zip
+      destinations:
+        - {{ platform }}/{{ locale }}/fennec-{{ version }}.{{ locale }}.android-arm.cppunittest.tests.zip
+    target.crashreporter-symbols-full.zip:
+      s3_key: fennec-{{ version }}.{{ locale }}.android-arm.crashreporter-symbols-full.zip
+      destinations:
+        - {{ platform }}/{{ locale }}/fennec-{{ version }}.{{ locale }}.android-arm.crashreporter-symbols-full.zip
+    target.crashreporter-symbols.zip:
+      s3_key: fennec-{{ version }}.{{ locale }}.android-arm.crashreporter-symbols.zip
+      destinations:
+        - {{ platform }}/{{ locale }}/fennec-{{ version }}.{{ locale }}.android-arm.crashreporter-symbols.zip
+    target.json:
+      s3_key: fennec-{{ version }}.{{ locale }}.android-arm.json
+      destinations:
+        - {{ platform }}/{{ locale }}/fennec-{{ version }}.{{ locale }}.android-arm.json
+    target.mochitest.tests.zip:
+      s3_key: fennec-{{ version }}.{{ locale }}.android-arm.mochitest.tests.zip
+      destinations:
+        - {{ platform }}/{{ locale }}/fennec-{{ version }}.{{ locale }}.android-arm.mochitest.tests.zip
+    target.mozinfo.json:
+      s3_key: fennec-{{ version }}.{{ locale }}.android-arm.mozinfo.json
+      destinations:
+        - {{ platform }}/{{ locale }}/fennec-{{ version }}.{{ locale }}.android-arm.mozinfo.json
+    target.reftest.tests.zip:
+      s3_key: fennec-{{ version }}.{{ locale }}.android-arm.reftest.tests.zip
+      destinations:
+        - {{ platform }}/{{ locale }}/fennec-{{ version }}.{{ locale }}.android-arm.reftest.tests.zip
+    target.talos.tests.zip:
+      s3_key: fennec-{{ version }}.{{ locale }}.android-arm.talos.tests.zip
+      destinations:
+        - {{ platform }}/{{ locale }}/fennec-{{ version }}.{{ locale }}.android-arm.talos.tests.zip
+    target.test_packages.json:
+      s3_key: fennec-{{ version }}.{{ locale }}.android-arm.test_packages.json
+      destinations:
+        - {{ platform }}/{{ locale }}/fennec-{{ version }}.{{ locale }}.android-arm.test_packages.json
+    target.txt:
+      s3_key: fennec-{{ version }}.{{ locale }}.android-arm.txt
+      destinations:
+        - {{ platform }}/{{ locale }}/fennec-{{ version }}.{{ locale }}.android-arm.txt
+    target.web-platform.tests.zip:
+      s3_key: fennec-{{ version }}.{{ locale }}.android-arm.web-platform.tests.zip
+      destinations:
+        - {{ platform }}/{{ locale }}/fennec-{{ version }}.{{ locale }}.android-arm.web-platform.tests.zip
+    target.xpcshell.tests.zip:
+      s3_key: fennec-{{ version }}.{{ locale }}.android-arm.xpcshell.tests.zip
+      destinations:
+        - {{ platform }}/{{ locale }}/fennec-{{ version }}.{{ locale }}.android-arm.xpcshell.tests.zip
+    target_info.txt:
```

Comment:
This file is the reason why I tried a different approach for the candidates templates. @escapewindow please let me know if you feel this is better or worse. This *same* file needs to be copied in three diferent places within the candidates bucket with three different namings. So far it's the only one but there could be others in the desktop part. Rather than adding `if file == target_info.yxy do: stuff od` within the code, I preferred to use this approach. What do you think?